frontend_lib/src/container/EmojiReactionButtons.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'react-i18next'
+
+import {
+  postContentReaction,
+  deleteContentReaction,
+  getContentReactionList
+} from '../action.async.js'
+
+import { sendGlobalFlashMessage, handleFetchResult } from '../helper.js'
+import { TracimComponent } from '../tracimComponent.js'
+
+import {
+  TLM_ENTITY_TYPE as TLM_ET,
+  TLM_CORE_EVENT_TYPE as TLM_CET
+} from '../tracimLiveMessage.js'
+
+import EmojiReactionButtonsComponent from '../component/EmojiReactions/EmojiReactionButtons.jsx'
+
+const ERR_REACTION_ALREADY_EXISTS = 3013
+const ERR_REACTION_NOT_FOUND = 1022
+
+class EmojiReactionButtons extends React.Component {
```

Comment:
Having two separate names for the component and the container would be more explicit IMO.

Reply:
I seem to recall @Skylsmoi is not a huge fan of suffixing by `Component`… would EmojiReactionArea and EmojiReactionButtons work?